ngraph/core/src/op/or.cpp

```diff
@@ -34,6 +34,13 @@ op::v1::LogicalOr::LogicalOr(const Output<Node>& arg0,
     constructor_validate_and_infer_types();
 }
 
+bool ngraph::op::v1::LogicalOr::visit_attributes(AttributeVisitor& visitor)
+{
+    NGRAPH_OP_SCOPE(v1_LogicalOr_visit_attributes);
+    BinaryElementwiseLogical::visit_attributes(visitor);
```

Comment:
Why we need to override visit_attributes if it is already implemented in base class?

Reply:
I propose to fix it in this PR.

Reply:
Corrected.